python-backend/app/commands.py

```diff
@@ -164,7 +165,7 @@ def delete_data():
             click.echo(f'Error, failed on commit.')
             raise
 
-    if app.config.get('ENVIRONMENT_NAME') == 'prod':
+    if app.config.get('ENVIRONMENT_NAME') == 'test' or app.config.get('ENVIRONMENT_NAME') == 'prod':
```

Comment:
Can you look at refactoring the scheduler initialization a little to ensure we dont keep this check all over the place?

Reply:
Removed as with the refactor there shouldn't be a problem here.